### ISSUE TITLE
500 server error due to search institutions with no search term

### ISSUE THREAD

**Issue by patrickvinograd** [department-of-veterans-affairs/gi-bill-comparison-tool#407]
Several 500 Internal Server Errors with this flow:

```
I, [2016-09-08T13:27:15.688059 #2401]  INFO -- : Started GET "/gi-bill-comparison-tool/institutions/search" for 152.131.36.97 at 2016-09-08 13:27:15 -0400
I, [2016-09-08T13:27:15.689280 #2401]  INFO -- : Processing by InstitutionsController#search as */*
I, [2016-09-08T13:27:15.690667 #2401]  INFO -- : Completed 500 Internal Server Error in 1ms (ActiveRecord: 0.0ms)
F, [2016-09-08T13:27:15.691206 #2401] FATAL -- :
NoMethodError (undefined method `empty?' for nil:NilClass):
  app/models/institution.rb:132:in `search'
  app/controllers/institutions_controller.rb:47:in `search'
```

Appears that the request has no params, so search_term is nil in models/institution.rb search method.

**Comment by patrickvinograd** [comment]
Easiest fix to avoid the 500 error is just change `search_term.empty?` to `search_term.blank?` in `app/models/institution.rb`. That winds up generating a results page with all 34k+ institutions, paginated. And with #410 it's pretty inefficient because it is doing a `select *` query on institutions. So, fix #410 first, then do the above fix.

**Comment by patrickvinograd** [comment]
#439 turned out to be a duplicate of this. Both fixed via #443.
